Packs/IPWhitelisting/Playbooks/playbook-IP_Whitelist_-_Okta_Zone.yml

```diff
@@ -0,0 +1,727 @@
+id: IP Whitelist - Okta Zone
```

Comment:
We are re-ordering packs for marketplace, everything "okta" should be in the okta pack. Same goes for the other vendor specific playbooks in the IPWhitelisting packs.